### PR TITLE
111 add resize

### PR DESCRIPTION
**Pull request by boxadesign**

### CODE REVIEW

**Comment by bameyrick** [comment]
Could you update all example components where the `useIframe` option is used?

File: src/components/timeout/index.njk [hunk 1]
```diff
@@ -7,6 +7,7 @@ title: Timeout
 This component may be deprecated as runner now does server side timeouts and this client side one is flaky.
 {{
     patternlibExample({
-        "path": "components/timeout/examples/timeout/index.njk"
+        "path": "components/timeout/examples/timeout/index.njk",
+        "iframeHeight": 500
```
Comment (bameyrick) [inline src/components/timeout/index.njk]: You missed one height param soz

Comment (boxadesign) [reply src/components/timeout/index.njk]: JB's fault

Comment (bameyrick) [reply src/components/timeout/index.njk]: kratz

File: package.json [hunk 2]
```diff
@@ -121,6 +121,7 @@
   "dependencies": {
     "a11y-dialog": "^5.2.0",
     "core-js": "^2.6.0",
+    "iframe-resizer": "^3.6.5",
```
Comment (bameyrick) [inline package.json]: Ideally this should be a dev dependency

File: src/styles/page-template/src/_template.njk [hunk 3]
```diff
@@ -50,6 +50,9 @@
         {% endblock %}
         {% block bodyEnd %}{% endblock %}
         <script type="module" src="/scripts/bundle.js"></script>
+        {% if isPatternLib %}
+            <script type="module" src="/scripts/patternlib.js"></script>
```
Comment (bameyrick) [inline src/styles/page-template/src/_template.njk]: The `nomodule` version of the patternlib script should also be included or the resizer wont work in IE11

**Comment by jrbarnes9** [comment]
[Add resize controls to patternlib example frames](https://trello.com/c/V589f8ta/111-add-resize-controls-to-patternlib-example-frames)

**Pull request merged**